source/extensions/filters/http/ext_proc/ext_proc.cc

```diff
@@ -766,6 +781,16 @@ void Filter::mergePerRouteConfig() {
     ENVOY_LOG(trace, "Setting new GrpcService from per-route configuration");
     grpc_service_ = *merged_config->grpcService();
   }
+
+  // Disable the filter if there is no valid GrpcService configuration to avoid possible runtime
+  // panics.
+  if (grpc_service_.target_specifier_case() ==
```

Comment:
If we add the check here, it looks to me we should set the client_ to be null pointer in case oneof is not set. So other places, like the above in openStream: 
stream_ = client_->start(*this, grpc_service_, decoder_callbacks_->streamInfo()); 
should just check whether client_ is nullptr and return

Reply:
This is removed now.